lib/utils/isStandardSyntaxHexColor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/**
+ * Check whether a hex color is standard
+ *
+ * @param {string} property
+ * @returns {boolean}
+ */
+module.exports = function (property) {
```

Comment:
[suggestion] Let's name the function for better debuggability:

```suggestion
module.exports = function isStandardSyntaxHexColor(property) {
```